ci/scripts/run-check_ci.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -eu
+
+#####################################################################################
+# Script description: script to check the status of an experiment as reported
+#                     by Rocoto
+#####################################################################################
+
+TEST_DIR=${1:-${TEST_DIR:-?}}  # Location of the root of the testing directory
+pslot=${2:-${pslot:-?}}        # Name of the experiment being tested by this script
+
+# TEST_DIR contains 2 directories;
+# 1. HOMEgfs: clone of the global-workflow
+# 2. RUNTESTS: A directory containing EXPDIR and COMROT for experiments
+# # e.g. $> tree ./TEST_DIR
+# ./TEST_DIR
+# ├── HOMEgfs
+# └── RUNTESTS
+#     ├── COMROT
+#     │   └── ${pslot}
+#     └── EXPDIR
+#         └── ${pslot}
+HOMEgfs="${TEST_DIR}/HOMEgfs"
+RUNTESTS="${TEST_DIR}/RUNTESTS"
+
+# Source modules and setup logging
+echo "Source modules."
+source "${HOMEgfs}/workflow/gw_setup.sh"
+
+# cd into the experiment directory
+echo "cd ${RUNTESTS}/EXPDIR/${pslot}"
+cd "${RUNTESTS}/EXPDIR/${pslot}" || (echo "FATAL ERROR: Unable to cd into '${RUNTESTS}/EXPDIR/${pslot}', ABORT!"; exit 1)
+
+# Name of the Rocoto XML and database files
+xml="${pslot}.xml"
+db="${pslot}.db"
+
+# Ensure the XML is present for the experiment
+if [[ ! -f "${xml}" ]]; then
+  echo "FATAL ERROR: XML file ${xml} not found in '${pslot}', experiment ${pslot} failed, ABORT!"
+  exit 1
+fi
+
+# Launch experiment
+echo "Launch experiment with Rocoto."
+rocotorun -v "${ROCOTO_VERBOSE:-0}" -w "${xml}" -d "${db}"
+sleep 30
```

Comment:
Are these sleeps necessary after `rocotorun`? I didn't think it returned until it was done. If they are needed, (a) I'm worried 30 may be too short (b) the length should be abstracted to a variable so we can change them all in one place.

Reply:
please don't over-engineer.